core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.html

```diff
@@ -41,18 +41,20 @@
           <span> <[skill.misconception_count]> </span>
         </a>
       </td>
-      <td ng-if="getMergeableSkillSummaries().length > 0">
-        <button class="btn btn-secondary protractor-test-merge-skills-button skills-list-action-button"
-                ng-click="mergeSkill(skill)">
-          <span> Merge Skill </span>
-        </button>
-      </td>
       <td ng-if="getEditableTopicSummaries().length > 0 && !isUnpublishedSkill()">
         <button class="btn btn-secondary skills-list-action-button
         protractor-test-assign-skill-to-topic-button" ng-click="assignSkillToTopic(skill.id)">
           <span> Assign to Topic </span>
         </button>
       </td>
+      <td class="merge-skill" ng-if="getMergeableSkillSummaries().length > 0">
```

Comment:
Can you maybe add a tooltip when the user hovers over the icon with the text 'Merge Skill'? Since, this may not be clear to all.

Reply:
Done! Added a title attribute.